input/otlp/metadata.go

```diff
@@ -194,6 +194,9 @@ func translateResourceMetadata(resource pcommon.Resource, out *modelpb.APMEvent)
 			if out.Host == nil {
 				out.Host = &modelpb.Host{}
 			}
+			if v.Type() != pcommon.ValueTypeSlice {
+				break
```

Comment:
```suggestion
				break // switch
```
nit: I'm always very nervous about `break` in switch case